Sort quarantine cards

LeonGeorgi opened the pull request.

LeonGeorgi committed:
Sort quarantine cards

Lodifice commented on back/multiplayer.py:
```diff
@@ -1,39 +1,27 @@
 import random
 
-cards = [ "Schell 7",
-          "Schell 8",
-          "Schell 9",
-          "Schell 10",
-          "Schell Unter",
-          "Schell Ober",
-          "Schell König",
-          "Schell Ass",
-          "Herz 7",
-          "Herz 8",
-          "Herz 9",
-          "Herz 10",
-          "Herz Unter",
-          "Herz Ober",
-          "Herz König",
-          "Herz Ass",
-          "Blatt 7",
-          "Blatt 8",
-          "Blatt 9",
-          "Blatt 10",
-          "Blatt Unter",
-          "Blatt Ober",
-          "Blatt König",
-          "Blatt Ass",
-          "Eichel 7",
-          "Eichel 8",
-          "Eichel 9",
-          "Eichel 10",
-          "Eichel Unter",
-          "Eichel Ober",
-          "Eichel König",
-          "Eichel Ass" ]
```
I really like this change.

Lodifice commented on back/multiplayer.py:
```diff
@@ -1,39 +1,27 @@
     random.seed(seed)
-    current_game_cards = random.sample(cards, k=len(cards))
-    return current_game_cards[player*10:player*10+10]
+    current_game_cards = random.sample(range(32), k=32)
+    player_cards = current_game_cards[player * 10:player * 10 + 10]
+    player_cards.sort()
+    return [f"{colors[card // 8]} {values[card % 8]}" for card in player_cards]
```
This builds three lists and a slice while the old code only built one list and a slice. Usage of sorted rather than list.sort is preferred.
Also the use of magic numbers became exorbitantly high, in addition to the 10s we also have 32s and 8s know.
Especially in case of the 8s some brain power has to be invested by the reader.
I suggest rewriting these lines  so that they are like the old ones, where the return value is wrapped in `sorted()`.
The lists `colors` and `values` can be elegantly combined to something like the old `cards` using a list comprehension and `itertools.product`.

LeonGeorgi committed:
Remove magic numbers

LeonGeorgi commented:
I understand that the magic numbers are not beautiful, but on the other hand the game "Skat" does have these magic numbers in the rules. However I still removed them.
I also replaced `list.sort()` with `sorted`, although I think it is a little more inefficient.

Actually I implemented the `itertools.product` solution first, but the sorting of the list was kind of ugly in this solution. That's why I find the idea to use just numbers from 0 to 31 really elegant. Is the mapping to the final string value really so difficult to understand?
